test/common/run-tests

```diff
@@ -188,8 +188,8 @@ def run(opts, image):
     if len(changed_tests) > 3:
         changed_tests = []
 
-    # HACK: Running all machines tests 3 times takes too long and the global timeout is reached
-    changed_tests = [test for test in changed_tests if "check-machines" not in test]
+    # HACK: Running all machines or realms tests 3 times takes too long and the global timeout is reached
+    changed_tests = [test for test in changed_tests if "check-machines" not in test and "check-realms" not in test]
```

Comment:
/ME is not huge fan of this... but oh well :man_shrugging:

Reply:
Right, it's a bit of "throws hands into the air" :crying_cat_face:  but at least we still have our flake db to point out remaining issues.